README.md

```diff
@@ -9,6 +9,13 @@ This project aims to make updating firmware on Linux automatic, safe and reliabl
 
 Additional information is available [at the website](https://fwupd.org/).
 
+## Where to get help?
+
+- Github issues & discussions in [this repository](https://github.com/fwupd/fwupd)
```

Comment:
I think it's GitHub, but other than that LGTM, thanks.

Reply:
The pre-commit hook fixed something in the README as well, will probably make the CI green again on `main`